workloads/jenkins/Jenkinsfile

```diff
@@ -83,6 +83,11 @@ pipeline {
             timeout 180 bash -x ./workloads/jenkins/scripts/test-dashboard.sh
           '''
 
+          echo "Test running a Deep Learning Example"
```

Comment:
How long does this test take? I've been trying to keep the PR Jenkinsfile light based on previous feedback.

Reply:
Good call on removing this -- it adds 11-12 minutes.